saleor/graphql/attribute/utils.py

```diff
@@ -335,23 +335,21 @@ def _validate_attributes_input(
 
         :raises ValidationError: when an invalid operation was found.
         """
-        variant_validation_pairs = None
+        variant_validation = False
```

Comment:
```suggestion
        variant_validation = True if is_variant else False
```

Reply:
This way we check `if is_variant` twice within this method. It is still preferred in the name of 1 line more?

Reply:
It's just a suggestion. We can leave it as it is.